src/main/java/uk/gov/hmcts/reform/iacaseapi/infrastructure/clients/CcdSupplementaryUpdater.java

```diff
@@ -53,6 +54,10 @@ public void setHmctsServiceIdSupplementary(final Callback<AsylumCase> callback)
         if (featureToggler.getValue("wa-R3-feature", false)) {
             requireNonNull(callback, "callback must not be null");
 
+            if (hasCitizenRole(userDetails.getRoles())) {
```

Comment:
Can we add journeyType as additional check to ensure update is excluded only AIP appeals with Citizen profiles?